### PR TITLE
Promoting Some Default Apache Logic: Rewrites and Env Config

### PR DESCRIPTION
**Pull request by jimdelois**
@bryanlatten 

Promoting a common patter from our implementations to the parent.

### CODE REVIEW

**Comment by bryanlatten** [comment]
@jimdelois looks great, can we add some quick notes and expectations on the readme, this is now making some big (and solid) assumptions for our specific use case

File: container/root/run.sh [hunk 1]
```diff
@@ -5,7 +5,27 @@ then
   echo "Running `composer install`"
   composer install
 else
-  echo "Starting apache2"
+
+  # Adds docker environment variables where Apache would normally eat them by
+  # entering them as Virtualhost environment variables
+  VARS=`env | grep ^CFG_`;
```
Comment (bryanlatten) [inline container/root/run.sh:11]: maybe we should use this (`CFG`) as its own environment variable?

**Comment by jimdelois** [comment]
@bryanlatten - Will do.  I'll update the README shortly.

File: container/root/etc/apache2/sites-available/000-docker.conf [hunk 2]
```diff
@@ -4,4 +4,15 @@
   LogLevel info
   ErrorLog /dev/stdout
   CustomLog /dev/stdout combined
+
+  <Directory /var/www/html>
+    # One rule to rule them all.
+    RewriteEngine On
+    RewriteBase /
+
+    RewriteCond %{REQUEST_FILENAME} !-f
```
Comment (bryanlatten) [inline container/root/etc/apache2/sites-available/000-docker.conf]: interestingly, if we do this, we can disable the options override -- granting us a "performance" boost, at the expense of some flexibility. what do you think?

File: container/root/run.sh (hunk 1 above)
Comment (jimdelois) [inline container/root/run.sh:11]: Will do as a follow-up PR!

**Pull request merged**